src/patternfly/components/CodeEditor/code-editor-container.hbs

```diff
@@ -0,0 +1,3 @@
+<div role="presentation" tabindex="0" class="{{pfv}}code-editor-container">
```

Comment:
This will update the markup to match the CSS class

```suggestion
<div role="presentation" tabindex="0" class="{{pfv}}code-editor__container">
```